homeassistant/components/camera/xiaomi.py

```diff
@@ -98,29 +101,34 @@ def get_latest_video_url(self):
             _LOGGER.warning("There don't appear to be any uploaded videos")
             return False
 
-        first_dir = dirs[-1]
-        try:
-            ftp.cwd(first_dir)
-        except error_perm as exc:
-            _LOGGER.error('Unable to find path: %s', first_dir)
-            _LOGGER.debug(exc)
-            return False
+        if self._model == 'xiaofang':
+            first_dir = dirs[-1]
+            try:
+                ftp.cwd(first_dir)
+            except error_perm as exc:
+                _LOGGER.error('Unable to find path: %s', first_dir)
+                _LOGGER.debug(exc)
```

Comment:
Don't double log

Reply:
I think both logging are different, 1st one is level 1 and next is level 2.

Reply:
yet both will log the same issue, just leave one of them

Reply:
Done code changes, how do I add a test?

Reply:
I do have both models and tested both and worked fine.

Reply:
Have a look at existing tests:

https://github.com/home-assistant/home-assistant/tree/dev/tests/components/camera

create a new file for your platform

Reply:
Do I need to create a test?. I don't see it was created for yi already. And it is just a few modification over yi which was already released.

Reply:
Yi camera had no options.

I would mock the FTP server, and validade the existence of the image file in a different path depending on the configured model.

Reply:
This camera has same options like Yi camera, and am not sure how to do testing, as I am not an expert in python. The code of this one is same as Yi Camera, except one directory above.